### PR TITLE
[IMP] web: added success_notify client action to display a notification

### PR DESCRIPTION
**Pull request by kebeclibre**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/static/src/js/framework/misc.js [hunk 1]
```diff
@@ -172,6 +172,12 @@ function ReloadContext (parent, action) {
 }
 core.action_registry.add("reload_context", ReloadContext);
 
+function SuccessNotify (parent, action) {
```
Comment (ged-odoo) [inline addons/web/static/src/js/framework/misc.js]: looking good, but you should add some docstring.  Here is a suggestion:

/**
 * This client action simply displays a notification. It is useful when we
 * do an action (such as a server action), that does not need to do anything,
 * except give some feedback to the user.
 *
 * @param {ActionManager} parent
 * @param {Object} action
 */

Comment (ged-odoo) [inline addons/web/static/src/js/framework/misc.js]: I think that the name implies a success, which is too restrictive.  What about NotifyUser?

**Comment by kebeclibre** [comment]
Changes have been made.
Also, the distinction between mandatory and optional params is dealt with.

File: addons/web/static/src/js/framework/misc.js [hunk 2]
```diff
@@ -172,6 +172,29 @@ function ReloadContext (parent, action) {
 }
 core.action_registry.add("reload_context", ReloadContext);
 
+/**
+ * This client action simply displays a notification. It is useful when we
+ * do an action (such as a server action), that does not need to do anything,
+ * except give some feedback to the user.
+ * @param {ActionManager} parent
+ * @param {Object} action
+ *
```
Comment (ged-odoo) [inline addons/web/static/src/js/framework/misc.js]: what we did elsewhere in master is to document sub parameters like this:

* @param {string} action.title
* @param {string} [action.message]
* @param {boolean} [action.sticky=false]

Note that optional parameters are in brackets, with optional value if it makes sense

**Pull request closed without merging**